test/python/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ if(${PARFLOW_HAVE_HYPRE})
     LW_var_dz_spinup
     terrain_following_grid_overland
     var_dz_1D
+    reservoir_mpi_test
```

Comment:
This doesn't look to be tested in parallel; doesn't look like it is testing under MPI.  Needs to be in the PARALLEL_ topo tests.